src/smdba/postgresqlgate.py

```diff
@@ -941,7 +941,7 @@ def do_system_check(self, *args, **params):
             changed = True
 
         # max_locks_per_transaction
-        if not conf.get('max_locks_per_transaction'):
+        if not conf.get('max_locks_per_transaction') or conf.get('max_locks_per_transaction') < '100':
```

Comment:
That's another problem. Ideally, I like your way best though, but then config should be more smart and do something like:
```python
def __setitem__(self, key, value):
    if self.get(key) != value:
        self[key] = value
        self.changed = True
```

Then:
```python
conf['foo'] = 'blah'
...
if conf.changed:
    ....
```